docs/installation.rst

```diff
@@ -71,19 +71,22 @@ If you are on Fedora, install the build dependencies with this command:
 
 .. code-block:: bash
 
-   sudo dnf install -y gcc make cmake bzip2-devel expat-devel file-devel glib2-devel libcurl-devel libmodulemd-devel libxml2-devel python3-devel python3-gobject python3-libmodulemd rpm-devel openssl-devel sqlite-devel xz-devel zchunk-devel zlib-devel
+   sudo dnf install -y gcc make cairo-devel cairo-gobject-devel cmake bzip2-devel expat-devel file-devel glib2-devel gobject-introspection-devel libcurl-devel libmodulemd-devel libxml2-devel openssl-devel python3-devel rpm-devel sqlite-devel xz-devel zchunk-devel zlib-devel
 
 If on CentOS or RHEL, use this commands:
 
-Ensure you have enabled ``epel`` repository
+Ensure you have enabled ``epel`` and ``powertools`` (in RHEL it is ``CodeReady``) repositories
```

Comment:
In CentOS 9 stream it's also called CRB

Reply:
I believe we do as e.g. in centos 8 it is needed for ``file-devel`` and ``libmodulemd-devel``

Reply:
I'm not sure why the package is in the core repos but the -devel package is in CRB, but you're right